packages/rocketchat-ui-flextab/flex-tab/tabs/membersList.coffee

```diff
@@ -64,6 +65,9 @@ Template.membersList.helpers
 		return '' unless roomData
 		return roomData.t in ['p', 'c'] and RocketChat.authz.hasAllPermission('add-user-to-room', this._id)
 
+	showUtcOffset: ->
+		return this.utcOffsetNum isnt Meteor.user().utcOffset
```

Comment:
@rodrigok would this make the helper reactive and add a lot of new computations?